tests/unit/src/kakarot/instructions/test_system_operations.cairo

```diff
@@ -115,10 +115,16 @@ func test__exec_revert{
     let ctx: model.ExecutionContext* = ExecutionContext.update_stack(ctx, stack);
 
     // When
-    SystemOperations.exec_revert(ctx);
+    let ctx = SystemOperations.exec_revert(ctx);
+    let is_reverted = ExecutionContext.is_reverted(ctx);
 
     // Then
-    // TODO: update test when revert does not break the execution
+    assert is_reverted = 1;
```

Comment:
as in `execute_at_address` we are still conveying the exception via a kakarot throw, for now.